ballista/scheduler/src/cluster/kv.rs

```diff
@@ -39,7 +39,6 @@ use ballista_core::serde::protobuf::{
 use ballista_core::serde::scheduler::{ExecutorData, ExecutorMetadata};
 use ballista_core::serde::BallistaCodec;
 use dashmap::DashMap;
-use datafusion::datasource::physical_plan::get_scan_files;
```

Comment:
The point of this PR is to remove this dependency (so we can remove it from DataFusion)